staking/programs/integrity-pool/src/state/pool.rs

```diff
@@ -165,7 +168,9 @@ impl PoolData {
         Ok(reward)
     }
 
+
     pub fn advance(&mut self, publisher_caps: &PublisherCaps, y: frac64) -> Result<()> {
+        let mut existing_publishers = [0_u8; MAX_CAPS / 8];
```

Comment:
tbh I feel like this should be it's own struct with methods to add a new element and check if an element is there already

Reply:
Done